src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/Dashboard.java

```diff
@@ -8,25 +18,75 @@
  */
 public class Dashboard extends JPanel {
 
-    private final int width = 250;
-    private final int height = 700;
-    private final int backgroundColor = 0x888888;
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final int WIDTH = 250;
+    private static final int HEIGHT = 700;
+    private static final int BACKGROUND_COLOR = 0x888888;
+
+    private Gui parent;
+
+    private TurnSignal leftTurnSignal;
+    private TurnSignal rightTurnSignal;
+    private TextSignal accSpeedSignal;
+    private TextSignal accDistanceSignal;
+    private TextSignal accOnOffSignal;
+    private TextSignal ppSignal;
+    private TextSignal lkaSignal;
+    private TextSignal lkaWarningSignal;
+    private TextSignal trafficSignSignal;
+    private TextSignal aebWarningSignal;
+    private TextSignal rrWarningSignal;
+    private DashboardText currentGear;
+    private DashboardText speedLimit;
+    private DashboardText steeringWheel;
+    private DashboardText xCoordinate;
+    private DashboardText yCoordinate;
+    private ProgressBars gasProgressBar;
+    private ProgressBars breakProgressBar;
+    private Gauge rpmGauge;
+    private Gauge kmhGauge;
 
-    Gui parent;
 
     private Thread timer = new Thread() {
-        int difference;
+        //int difference;    // ez nem tudom mire valo, egyelore maradjon
```

Comment:
Legjobb komment. :D 
Ha marad, akkor miért lett kikommentezve?

Reply:
Ezt még ti tettétek ide. Először nem töröltük mert hátha kell majd, aztán elfelejtettük kivenni, törölhető.
Viszont hogy csináljuk? Nyissunk meg egy új branch-et a team1-ből, majd merge-eljük a team1-be? Mert push-olni közvetlenül nem enged a team1-be. Vagy van vmi egyszerűbb lehetőség?

Reply:
Lehet nem voltatok múltkor, akkor is előkerült, hogy ha a forrás branch frissül, akkor a PR is frissül. Tehát elég a feature branchre tenni a javítást.

Reply:
De itt most a team1 a forrás branch, és nem egy feature branch. A team1-re pedig nem enged push-olni.
Azóta megcsináltam az észrevételek kezelését/javítását egy új branchben: team1-fix-pullreq52.
Csak az a baj, hogy a pull request amivel azt mergel-ni akartam a team1-be, elakadt. Írtam róla külön email-t neked, a CI check sárga már órák óta, ez az:

Reply:
bocs, submit-olt, folyt.:
szóval ez az:
Removed commented code, using dummy values #53
Van egy másik is a Team3-tól ami ugyanúgy beragadt a CI check-nél.